pyproject.toml

```diff
@@ -25,40 +25,40 @@ dependencies = [
     "django-fsm~=2.8.2",
     "django-heroku~=0.3.1",
     "django-hijack~=3.7.0",
-    "django-htmx~=1.17",
+    "django-htmx~=1.23.0",
     "django-nh3~=0.1.1",
     "django-pagedown~=2.2.1",
     "django-ratelimit~=4.1.0",
     "django-role-permissions~=3.2.0",
     "django-slack~=5.19.0",
-    "django-storages~=1.14.2",
-    "django-tables2~=2.7.0",
+    "django-storages~=1.14.5",
+    "django-tables2~=2.7.5",
     "django-tinymce~=4.1.0",
     "django-two-factor-auth~=1.17.0",
     "django-web-components~=0.2.0",
     "django~=4.2.0",
-    "django-stubs~=4.2.0",
+    "django-stubs~=5.1.3",
```

Comment:
ideally this should follow the version of Django

Reply:
Fixed